src/SpotifyApi.NetCore/LibraryApi.cs

```diff
@@ -0,0 +1,410 @@
+        {
+        }
+
+        public LibraryApi(HttpClient httpClient) : base(httpClient)
+        {
+        }
+        #endregion
+
+        #region CheckUserSavedAlbums
+        /// <summary>
+        /// Check if one or more albums is already saved in the current Spotify user’s ‘Your Music’ library.
+        /// </summary>
+        /// <param name="albumIds">Required. A comma-separated list of the Spotify IDs for the albums. Minimum: 1 ID. Maximum: 50 IDs.</param>
+        /// <returns>bool[] an array of true or false values, in the same order in which the ids were specified.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/library/check-users-saved-albums/
+        /// </remarks>
+        public async Task<bool[]> CheckUserSavedAlbums(
+            string[] albumIds,
+            string accessToken = null
+            ) => await CheckUserSavedAlbums<bool[]>(albumIds, accessToken);
+
+        /// <summary>
+        /// Check if one or more albums is already saved in the current Spotify user’s ‘Your Music’ library.
+        /// </summary>
+        /// <param name="albumIds">Required. A comma-separated list of the Spotify IDs for the albums. Minimum: 1 ID. Maximum: 50 IDs.</param>
+        /// <returns>bool[] an array of true or false values, in the same order in which the ids were specified.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/library/check-users-saved-albums/
+        /// </remarks>
+        public async Task<T> CheckUserSavedAlbums<T>(
+            string[] albumIds,
+            string accessToken = null
+            )
+        {
+            if (albumIds?.Length < 1 || albumIds?.Length > 50) throw new
+                    ArgumentException("A minimum of 1 and a maximum of 50 album ids can be sent.");
+
+            var builder = new UriBuilder($"{BaseUrl}/me/albums/contains");
+            builder.AppendToQueryAsCsv("ids", albumIds);
+            return await GetModel<T>(builder.Uri, accessToken);
+        }
+        #endregion
+
+        #region CheckUserSavedShows
+        /// <summary>
+        /// Check if one or more shows is already saved in the current Spotify user’s library.
+        /// </summary>
+        /// <param name="showIds">Required. A comma-separated list of the Spotify IDs for the shows. Minimum: 1 ID. Maximum: 50 IDs.</param>
+        /// <returns>bool[] an array of true or false values, in the same order in which the ids were specified.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/library/check-users-saved-shows/
+        /// </remarks>
+        public async Task<bool[]> CheckUserSavedShows(
+            string[] showIds,
+            string accessToken = null
+            ) => await CheckUserSavedShows<bool[]>(showIds, accessToken);
```

Comment:
* Method name should be `CheckUsersSavedShows`
* Remove `async` and `await` keywords here.